docs/approach.rst

```diff
@@ -206,22 +206,28 @@ Specifically, ``tedana`` offers two different approaches that perform this step.
 The simplest approach (the default ``mdl``, ``aic`` and ``kic`` options for
 ``--tedpca``) is based on a moving average (stationary Gaussian) process
 proposed by `Li et al (2007)`_.
-A moving average process is the output of a linear system (which in this case is
+A moving average process is the output of a linear system (which, in this case, is
 a smoothing filter) that has an independent and identically distributed
 Gaussian process as the input.
-If we assume that the linear system is shift invariant, the moving average
-process is a stationary Gaussian random process.
 Simply put, this process more optimally selects the number of components for
 fMRI data following a subsampling scheme described in `Li et al (2007)`_.
-The selection of components is performed with either of the three options
-provided by ``--tedpca``:
 
-* ``aic``: the Akaike Information Criterion, which is the least aggressive option;
+The amount of selected principal components depends on the selection criteria.
+For this PCA method in particular, ``--tedpca`` provides three different options
+to select the PCA components based on three widely-used model selection criterion:
+
+* ``aic``: the Akaike Information Criterion (`AIC`_), which is the least aggressive option;
   i.e., returns the largest number of components.
-* ``kic``: the Kullback-Leibler Information Criterion, which stands in the
-  middle in terms of aggressiveness.
-* ``mdl``: the Minimum Description Length, which is the most aggressive (and
-  recommended) option.
+* ``kic``: the Kullback-Leibler Information Criterion (`KIC`_), which stands in the
+  middle in terms of aggressiveness. You can see how KIC is related to AIC `here`_.
+* ``mdl``: the Minimum Description Length (`MDL`_), which is the most aggressive option;
+  i.e. returns the least number of components. This option is the **default and recommeded**
+  as we have seen it yields the most reasonable results.
+
+.. note::
+    Please, bear in mind that this is a data-driven dimensionality reduction approach. The default
+    option ``mdl`` might not yield perfect results on your data. We suggest you explore the ``kic``
+    and ``aic`` options if running ``tedana`` with ``mdl`` returns less components than expected.
 
 A more complicated approach involves applying a decision tree (similar to the
```

Comment:
Could you add something like, "In addition to the moving average process-based approaches described above, we also support a decision tree-based selection method. This method, `kundu`, ..."